template/build/webpack.base.conf.js

```diff
@@ -31,76 +31,73 @@ module.exports = {
       'components': path.resolve(__dirname, '../src/components')
     }
   },
-  resolveLoader: {
-    fallback: [path.join(__dirname, '../node_modules')]
-  },
   module: {
+    rules: [
     {{#lint}}
-    preLoaders: [
       {
         test: /\.vue$/,
-        loader: 'eslint',
+        loader: 'eslint-loader',
+        enforce: 'pre',
         include: [
           path.join(projectRoot, 'src')
         ],
         exclude: /node_modules/
       },
       {
         test: /\.js$/,
-        loader: 'eslint',
+        loader: 'eslint-loader',
+        enforce: 'pre',
         include: [
           path.join(projectRoot, 'src')
         ],
         exclude: /node_modules/
-      }
-    ],
+      },
     {{/lint}}
-    loaders: [
       {
         test: /\.vue$/,
-        loader: 'vue'
+        loader: 'vue-loader'
       },
       {
         test: /\.js$/,
-        loader: 'babel',
+        loader: 'babel-loader',
         include: [
           path.join(projectRoot, 'src')
         ],
         exclude: /node_modules/
       },
       {
-        test: /\.json$/,
-        loader: 'json'
-      },
-      {
         test: /\.(png|jpe?g|gif|svg)(\?.*)?$/,
-        loader: 'url',
-        query: {
+        loader: 'url-loader',
+        options: {
           limit: 10000,
           name: utils.assetsPath('img/[name].[hash:7].[ext]')
         }
       },
       {
         test: /\.(woff2?|eot|ttf|otf)(\?.*)?$/,
-        loader: 'url',
-        query: {
+        loader: 'url-loader',
+        options: {
           limit: 10000,
           name: utils.assetsPath('fonts/[name].[hash:7].[ext]')
         }
       }
     ]
   },
-  {{#lint}}
-  eslint: {
-    formatter: require('eslint-friendly-formatter')
-  },
-  {{/lint}}
-  vue: {
-    loaders: utils.cssLoaders({ sourceMap: useCssSourceMap }),
-    postcss: [
-      require('autoprefixer')({
-        browsers: ['last 2 versions']
-      })
-    ]
-  }
+  plugins: [new webpack.LoaderOptionsPlugin({
+    options: {
+      {{#lint}}
+      eslint: {
+        formatter: require('eslint-friendly-formatter')
+      },
+      {{/lint}}
+      vue: {
```

Comment:
We don't have to use LoaderOptionsPlugin with vue-loader, you can specify these options in the loader rule:

https://vue-loader.vuejs.org/en/configurations/advanced.html (scroll down to Werbpack 2 section)